chore: document sidebarTitle frontmatter (#3482)
shuding/nextra: docs/pages/docs/docs-theme/theme-configuration.mdx

<Screenshot src={sidebarTitleComponentImage} alt="Customized Sidebar" />

#### Customize Sidebar with Frontmatter

In addition, you can customize the sidebar title using the `sidebarTitle` property in your Frontmatter:

```md filename="pages/getting-started.mdx"
---
sidebarTitle: Getting Started 🚀
---
```
The priority of the sidebar title is as follows:

1. `title` property from `_meta.js` file
1. Frontmatter `sidebarTitle`
1. Frontmatter `title`

> 1. Filename of the page formatted with with [Title](https://title.sh) 

Reviewer: ```suggestion
1. Filename of the page formatted with [Title](https://title.sh) 
```

<Screenshot src={sidebarTitleComponentImage} alt="Customized Sidebar" />

> #### Customize Sidebar with Frontmatter

Reviewer: ```suggestion
### Customize Sidebar with Frontmatter
```

Reviewer: looks better like this 
![Uploading image.png…]()

In addition, you can customize the sidebar title using the `sidebarTitle` property in your front matter:

> ```md filename="pages/getting-started.mdx"

Reviewer: ````suggestion
```mdx filename="pages/getting-started.mdx"
````

to have mdx icon

> In addition, you can customize the sidebar title using the `sidebarTitle` property in your Frontmatter:
> 
> ```md filename="pages/getting-started.mdx"
> ---
> sidebarTitle: Getting Started 🚀
> ---
> ```
> 
> The priority of the sidebar title is as follows:
> 
> 1. `title` property from `_meta.js` file
> 1. Frontmatter `sidebarTitle`
> 1. Frontmatter `title`

Reviewer: ```suggestion
In addition, you can customize the sidebar title using the `sidebarTitle` property in your front matter:

```md filename="pages/getting-started.mdx"
---
sidebarTitle: Getting Started 🚀
---
```

The priority of the sidebar title is as follows:

1. `title` property from `_meta.js` file
1. Front matter `sidebarTitle`
1. Front matter `title`
```

let's stays with front matter https://github.com/jekyll/jekyll/issues/2396#issuecomment-50858507
